WordPress/src/main/java/org/wordpress/android/ui/stats/StatsUtils.java

```diff
@@ -411,12 +411,17 @@ public static synchronized Serializable parseResponse(StatsService.StatsEndpoint
         return model;
     }
 
-    public static void openPostInReaderOrInAppWebview(Context ctx, final PostModel post) {
-        final String postType = post.getPostType();
-        final String url = post.getUrl();
-        final long blogID = Long.parseLong(post.getBlogID());
-        final long itemID = Long.parseLong(post.getItemID());
-        if (postType.equals("post") || postType.equals("page")) {
+    public static void openPostInReaderOrInAppWebview(Context ctx, final String remoteBlogID,
+                                                      final String remoteItemID,
+                                                      final String itemType,
+                                                      final String itemURL) {
+        final long blogID = Long.parseLong(remoteBlogID);
+        final long itemID = Long.parseLong(remoteItemID);
+        if (itemType == null) {
+            // If we don't know the type of the item, open it with the browser.
+            AppLog.d(AppLog.T.UTILS, "Type of the item is null. Opening it in the in-app browser: " + itemURL);
+            WPWebViewActivity.openURL(ctx, itemURL);
+        } else if (itemType.equals("post") || itemType.equals("page")) {
```

Comment:
I've noticed "post" and "page" hard-coded in several places in stats. Can these be changed to constants?